src/interpreter/plugin/3rdparty/bessel/bessel.ts

```diff
@@ -0,0 +1,441 @@
+/* eslint-disable */
+/* bessel.js (C) 2013-present SheetJS -- http://sheetjs.com */
+/**
+ * @license
+ *                                  Apache License
+ Version 2.0, January 2004
+ http://www.apache.org/licenses/
```

Comment:
Small formatting fix
```suggestion
/**
 * @license
 bessel.js (C) 2013-present SheetJS -- http://sheetjs.com
 
                                 Apache License
                           Version 2.0, January 2004
                        http://www.apache.org/licenses/
```